From 8447b36e09fd2b4a420892267c91993e77299f4b Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Thu, 18 Feb 2016 13:21:59 +0100
Subject: [PATCH] man: ip-address: align synopsis with help output

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1231898
Upstream Status: iproute2.git commit 0b17394087a2c

commit 0b17394087a2c1b5564353bd21eb59f468130553
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Sep 18 20:17:00 2015 +0200

    man: ip-address: align synopsis with help output

    When fixing the BNF syntax error, I overlooked that 'ip address help'
    prints a more correct synopsis. This patch aligns them.

    Fixes: 715296b ("ip-address.8.in: fix BNF syntax error")
    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/ip-address.8.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man8/ip-address.8.in b/man/man8/ip-address.8.in
index f1efc43..43611b9 100644
--- a/man/man8/ip-address.8.in
+++ b/man/man8/ip-address.8.in
@@ -60,7 +60,7 @@ ip-address \- protocol address management
 .IR FLAG " := "
 .RB "[ " permanent " | " dynamic " | " secondary " | " primary " | \
 [ - ] " tentative " | [ - ] " deprecated " | [ - ] " dadfailed " | "\
-temporary " ] [ " CONFFLAG-LIST " ]"
+temporary " | " CONFFLAG-LIST " ]"
 
 .ti -8
 .IR CONFFLAG-LIST " := [ "  CONFFLAG-LIST " ] " CONFFLAG
-- 
1.8.3.1

